code/game/machinery/magnet.dm

```diff
@@ -21,7 +21,7 @@
 	var/code = 0 // frequency code, they should be different unless you have a group of magnets working together or something
 	var/turf/center // the center of magnetic attraction
 	var/on = FALSE
-	var/pulling = 0
+	var/magneting = 0

```

Comment:
You can reverse this change since the reason this was done is gone from this pr now.